docs/installation.md

```diff
@@ -0,0 +1,79 @@
+## Installation
+This guide uses minishift version v1.33.0+ as the local Kubernetes cluster
+and quay.io for the public registry. We also test on minikube version v0.35+.
+`kubectl` is used but can be substituted* with `oc`.
+
+### Supported versions
+* [OKD](https://www.okd.io/)
+  * Experimental: 3.11
+* [OpenShift® Container Platform](https://www.openshift.com/products/container-platform/)
+  * Fully supported: 4.0
+  * Experimental: 3.11
+* [kubernetes](https://kubernetes.io/)
+  * Experimental: 1.11-1.13
+
+Note: Experimental tag refers to some workloads that might be functioning
+
+### Requirements
+<!---
+TODO(aakarsh):
+Get the specific versions for requirements
+-->
+
+The main requirements are as follows:
+* Running Kubernetes cluster - [supported versions](#Supported-Versions)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [git](https://git-scm.com/downloads)
+
+Note: Please login as admin user
+
+The following requirements are needed to make/test changes to operator:
+* [operator-sdk](https://github.com/operator-framework/operator-sdk)
+* [docker](https://docs.docker.com/install/)
+
+Note: You also require a [quay](https://quay.io/) account to push images
+
+The following optional requirements are needed if using OKD/OCP < 4.0:
+* [dep](https://golang.github.io/dep/docs/installation.html)
+* [operator-sdk](https://github.com/operator-framework/operator-sdk)
+* [go](https://golang.org/dl/)
+* [OLM](https://github.com/operator-framework/operator-lifecycle-manager)
+
+The workloads could also have their own requirements which would be specified
+in the installation guide.
+
+### Deploying operator
+Note: Ideally you'd want to clone to your go path
```

Comment:
hm, I don't think this is necessary, as long as they have their gopath setup properly for the operator-sdk, no need to build / write code in their $GOPATH.